SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -857,6 +865,44 @@ FString FSpatialGDKEditorToolbarModule::GetOptionalExposedRuntimeIP() const
 	}
 }
 
+void FSpatialGDKEditorToolbarModule::UpdateMobileClient() const
+{
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	const UIOSRuntimeSettings* IOSRuntimeSettings = GetDefault<UIOSRuntimeSettings>();
+	const FString ProjectName = FApp::GetProjectName();
+
+	const FString CommandLineArgs = FString::Printf(TEXT( "../../../%s/%s.uproject %s -game -log -workerType UnrealClient"), *ProjectName, *ProjectName, *(SpatialGDKSettings->ExposedRuntimeIP));
+	const FString Filename = FPaths::Combine(*FPaths::ProjectLogDir(), TEXT("ue4commandline.txt"));
+	if(!FFileHelper::SaveStringToFile(CommandLineArgs, *Filename, FFileHelper::EEncodingOptions::ForceUTF8WithoutBOM))
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("Failed to write command line args to file: %s"), *Filename);
```

Comment:
```suggestion
		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("UpdateMobileClient: Failed to write command line args to file: %s"), *Filename);
```
Let's be more specific about what's failing here.